docroot/modules/custom/layout_builder_custom/layout_builder_custom.module

```diff
@@ -450,6 +471,13 @@ function _layout_builder_custom_block_validate(array &$form, FormStateInterface
   /** @var \Drupal\layout_builder\SectionComponent $component */
   $component = $form_state->getFormObject()->getCurrentComponent();
 
+  // Run unique_id clean for when the AJAX callback is not triggered.
+  if ($form_state->getValue(['unique_id_wrapper', 'unique_id'])) {
+    _layout_builder_custom_unique_id($form, $form_state);
```

Comment:
Seems like we could just run this through `Html::cleanCssIdentifier`. The ajax callback was specifically for the form element. Maybe something like: 
```
if ($unique_id = $form_state->getValue(['unique_id_wrapper', 'unique_id'])) {
  $form_state->setValue(['unique_id_wrapper', 'unique_id'], Html::cleanCssIdentifier($unique_id));
}

Reply:
Good call